app/api/events.py

```diff
@@ -107,7 +108,13 @@ class Meta:
 
 
 class EventList(ResourceList):
-    decorators = (jwt_required, )
+    def query(self, view_kwargs):
+        query_ = self.session.query(Track)
+        if view_kwargs.get('user_id') is not None:
+            query_ = query_.join(User).filter(User.id == view_kwargs['user_id'])
+        return query_
+
+    decorators = (jwt_required, accessible_events, )
```

Comment:
Just a few questions/opinions. I am not very sure about this so asking. Firstly, why not only accessible_events? Secondly, if you are giving decorators accessible_events, what does it mean for a POST request? Or does it only apply for a GET?